vmpooler.yaml.aliasedpools

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Is this test fixture file in the right location? i.e. the root of the project?  Should it be under `/spec/`?

Reply:
It's not used by the spec tests, in fact I have not seen any `.yaml` file in the spec directory. I want that example dummy config file in the repo to share that file with my team members working on integrating this new information into the ABS scheduler. I think the directory `ex` would be better for this?

Reply:
Perhaps the `docs` dir?